### PR TITLE
Get all secrets from org

### PR DESCRIPTION
**Pull request by gerardsegarra**
Get secrets from organization

### CODE REVIEW

**Comment by gerardsegarra** [comment]
We weren't planning on adding anything else. Do you have any suggestion?

**Comment by gerardsegarra** [comment]
@qua-jones is this mergeable?

**Comment by shlokabhalgat** [comment]
Hello folks, will we be having this one rolling out? I was working on a project that requires this, so was hoping we could add this one. :)

File: github/Organization.py [hunk 1]
```diff
@@ -673,7 +688,7 @@ def create_secret(
         status, headers, data = self._requester.requestJson(
             "PUT", f"{self.url}/actions/secrets/{secret_name}", input=put_parameters
         )
-        return status == 201
+        return status <= 299
```
Comment (EnricoMi) [inline github/Organization.py:691]: @gerardsegarra can you motivate this change, please?

Comment (EnricoMi) [reply github/Organization.py:691]: Looks like `201` is used when creating a secret and `204` when updating a secret (calling `create_secret` on an existing secret): https://github.com/PyGithub/PyGithub/pull/2218#issue-1201786009

@jonashartwig is that your observation?

https://github.com/PyGithub/PyGithub/pull/2218/files#diff-0ccc1045a7c37896a11edf884024adbf4b46fe9c8c8b05ed2e5e64b01dea5225R1490

@nthings what do you think? Should `create_secret` fail when the secret exists already, though return status is 2xx?

I think, updating a secret should be done only through `get_secret().edit()`.

Comment (jonashartwig) [reply github/Organization.py:691]: yes, it is.

**Pull request closed without merging**